api/types/constants.go

```diff
@@ -341,6 +341,10 @@ const (
 	// KindHeadlessAuthentication is a headless authentication resource.
 	KindHeadlessAuthentication = "headless_authentication"
 
+	// KindAssistant is a Teleport Assist resource kind. It's used by
+	// Teleport Assist to store its objects as messages and conversations.
+	KindAssistant = "assistant"
```

Comment:
So is this a singleton resource or can there be multiple assistants in a single Teleport cluster?

Reply:
Sorry, misleading comment. This constant is used in RBAC to control access to assistant-related resources.

Reply:
Ok, since it's not a role I would suggest this then.

```
// KindAssistant is used to program RBAC for
// Teleport Assist resources.
```